### DISCUSSION TITLE
Why does my academic affiliation not verify even though I uploaded official materials?

### DISCUSSION THREAD

**Discussion by Lily-Le** [community/community#53373] [GitHub Education] [answered]
I've tried 16 times.  I uploaded a bunch of materials like my student ID card, certification provided on our school website,  printed studying certificate with official seals, etc.  My classmate used the printed studying certificate and got proven the first time. Why is it that mine is always rejected? 
What's wrong with me? I'm a bit frustrated.

**Comment by samus-aran** [comment]
Hi @Lily-Le  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by Lily-Le** [comment, marked as answer]
Thanks! @samus-aran 
 Problem solved. I took a photo of both my studying certificate and student ID card. Then I tried two more times and finally succeeded! 
At first, my rejection letter contained some detailed reasons for my rejection, such as campus location and user profile. 
But after I've solved the problems listed before,  the last few rejection letters just told me 'Unfortunately, we were unable to verify your academic affiliation based on the information you submitted' without other more detailed reasons, and made me confused.
Then I tried a few more times with the same materials and got succeeded. Hope this can help some other people facing the same problem.

**Reply by samus-aran** [reply]
Thank you for coming back @Lily-Le and providing your solution 👍  What will you be exploring first now you've been verified?

**Reply by Lily-Le** [reply]
Copilot! Since my research area involves a lot of coding, I'm really looking forward to how it can facilitate my coding experience.  Actually, I've already tried Copilot, and I can definitely say it's smarter than some common code completion plugins. I'm exploring further usage of copilot and other education benefits now.
